src/main/java/uk/gov/digital/ho/hocs/casework/api/StageService.java

```diff
@@ -529,7 +549,7 @@ private void updateStages(Set<StageWithCaseData> stages) {
         for (StageWithCaseData stage : stages) {
             stagePriorityCalculator.updatePriority(stage, stage.getCaseDataType());
             daysElapsedCalculator.updateDaysElapsed(stage.getData(), stage.getCaseDataType());
-            stage.setTag(stageTagsDecorator.decorateTags(stage.getData(), stage.getStageType()));
+            //stage.setTag(stageTagsDecorator.decorateTags(stage.getData(), stage.getStageType()));
```

Comment:
No longer required as retrieved from the table.

Reply:
For the time being this has been reinstated but integrated into the functionality.